src/main/java/command/CommandMarkUnmarkHandler.java

```diff
@@ -0,0 +1,24 @@
+package command;
+import main.Main;
```

Comment:
```suggestion
package command;

import main.Main;
```
Should the import statement directly follow the package statement?